__tests__/app.test.js

```diff
@@ -125,4 +125,54 @@ describe("GET /api/articles with comment_count key and value",()=>{
   })
   
 })
+describe("GET /api/articles/:article_id/comments", ()=>{
+  test("200: responds with an array of comments for the given article_id ",()=>{
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then(({body: {comments}})=>{
+
+      expect(comments.length).toBe(11)
+      comments.forEach((comment)=>{
+
+        expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number)
+        })
+      })
+    })
+    
+ })
+//  test('200: responds with an empty array if article exists but has no comments', () => {
+//   return request(app)
+//     .get('/api/articles/12/comments')
+//     .expect(200)
+//     .then(({body: {comments}}) => {
+//       console.log(comments)
+//       expect(comments).toEqual([]);
+//     });
+// });
```

Comment:
This test is something that we want. See if you can get it to pass. You'll want to do some sort of validation to check that it's an article that exists first 👍